messaging/src/test/java/org/axonframework/commandhandling/distributed/ConsistentHashTest.java

```diff
@@ -109,4 +110,14 @@ void testConflictingHashesDoNotImpactMembership() {
         assertEquals(member1.name(), consistentHash.getMembers().iterator().next().name());
         assertEquals(consistentHash.getMembers(), consistentHashModified.getMembers());
     }
+
+    @Test
+    void testEquals1() throws Exception {
```

Comment:
I would expect the name of the test method to resemble in a bit more detail what's expected.
So more of a scenario description than just "equals test one".

For example:

```suggestion
    void testEqualsIsFalseForMatchingModifiedInstanceWithDefaultInstance() throws Exception {
```